dev/README.md

```diff
@@ -179,3 +340,26 @@ The script runs forever, on linux machines press `CTRL+C` to stop it (or just se
 With the parameters `--manager`, `--keycloak`, `--username` and `--password` you can configure the script so to target any environment (for example the demo environment).
 
 The grafana dashboards in the `grafana` folder are just for development purposes, but it's good to keep them in sync with the dashboards deployed in the demo environment (those dashboards are located at `kustomize/overlays/prod/observability/grafana`).
+
+## Binaries migration guide (Apr 22)
+
+## Renamed scripts
+
+Some scripts from the `bin` folder have been renamed for more coherence.  
+Here is the list of new names:
+
+| Old name                           | New name                     |
+|:-----------------------------------|------------------------------|
+| `shard-run.sh`                     | `local-shard-start.sh`       |
+| `manager-run.sh`                   | `local-manager-start.sh`     |
+| `minikube-start.sh`                | `cluster-start.sh`           |
+| `minikube-build-docker-images.sh`  | `cluster-load-containers.sh` |
+
+## Config files moved
+
+On the first execution of the `bin/common.sh` script, configuration files will be automatically transferred to the `config` folder.  
+Here are the moves done:
```

Comment:
What is the recommended way for this migration. In past , I have never run `bin/common.sh` on its own. Is it the same? If not, we should explicitly mention, that it is first step of migration.

Reply:
this script is a utils script which is called from all other scripts. You should not need to run it.
Here is maybe a better sentence. Let me know if you are ok with that @jomarko 

```suggestion
On the execution of any script, it should call the `bin/common.sh` script, which will automatically transfer the configuration files to the `config` folder.  
Here are the moves done:
```

Reply:
that sounds better to me @radtriste , yes